### ISSUE TITLE
[QUERY] Azure Communication Service .Net SDK: what does AddTextAnalyticsClient() in Call.cs do?

### ISSUE THREAD

**Issue by h0las-dev** [Azure/azure-sdk-for-net#27047]
### Library name and version

Azure.Communication.Calling 1.0.0-beta.29

### Query/Question

Hi! I found this method in Call.cs class and I try to realize what it does. I expext there is feature to analyze text transcript during video call, it is correct? Can anyone share some code examples or just explain ho to use it? I add my analysis client and what next? How I will get response with analysis from client during video call? Thank you!

### Environment

Windows 10 OS Build 17763.2565
Microsoft.NETCore.UniversalWindowsPlatform version 6.2.13
Microsoft Visual Studio Community 2022 (64-bit) Version 17.0.4

**Comment by SaurabhSharma-MSFT** [comment]
@h0las-dev Which `call.cs` you are referring to? Can you please share the file path ?

**Comment by h0las-dev** [comment]
@SaurabhSharma-MSFT yes, it's Call.cs from Azure.Communication.Calling namespace in Azure.Communication.Calling SDK

**Comment by RyanHill-MSFT** [comment]
@h0las-dev I responded over on https://docs.microsoft.com/en-us/answers/questions/737970/index.html.

TL;DR AddTextAnalyticsClient() I found as an extension method in Azure.AI.Analytics and it didn't match any signatures on the Call object. If your code is GitHub, we can take a closer look.

**Comment by h0las-dev** [comment]
Oh, I'm really sorry, it was my fault, I didn't check it's just extension. Thank you!
